frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -411,7 +411,29 @@ class Chart extends Component<Props, State> {
     });
   };
 
-
+  private relativeTime = new Intl.RelativeTimeFormat(this.props.i18n.language, { style: 'long' });
+
+  private renderRatesTimestamps = (ratesTimestamps: { [key in CoinCode]: number }) => {
+    const { t } = this.props;
+    const outdatedCoins = Object.keys(ratesTimestamps);
+    // can be either "COIN", "COIN and COIN" or "COIN, COIN and COIN"
+    const coinOrCoinsWithAnd =
+      (outdatedCoins.length >= 2)
+        ? [
+          outdatedCoins.slice(0, -1).map(coin => coin.toUpperCase()).join(', '),
+          t('generic.and'),
+          outdatedCoins.at(-1)?.toUpperCase()
+        ].join(' ')
+        : outdatedCoins[0].toUpperCase();
+
+    const oldestTimestamp = Math.min(...Object.values(ratesTimestamps));
+    const hoursAgo = Math.ceil((oldestTimestamp - Date.now()) / 3600000);
+
+    return t('chart.dataOldTimestamps', {
+      name: coinOrCoinsWithAnd,
+      timeAgo: this.relativeTime.format(hoursAgo, 'hours'),
```

Comment:
FYI, timeAgo will now always be translated to whatever `this.props.i18n.language` is set, regarding of if there is `chart.dataOldTimestamps` translation or not. so ideally we have `chart.dataOldTimestamps` translation in every lang.